test/tstQueryTreeComparisonWithBoost.cpp

```diff
@@ -0,0 +1,174 @@
+/****************************************************************************
+ * Copyright (c) 2012-2020 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "ArborX_BoostRTreeHelpers.hpp"
+#include "ArborX_EnableDeviceTypes.hpp" // ARBORX_DEVICE_TYPES
+#include <ArborX_LinearBVH.hpp>
+
+#include <boost/test/unit_test.hpp>
+
+#include <array>
+#include <functional>
+#include <iostream>
+#include <random>
+#include <vector>
+
+#include "Search_UnitTestHelpers.hpp"
+
+BOOST_AUTO_TEST_SUITE(ComparisonWithBoostGeometryIndex)
```

Comment:
Probably a good idea to match the filename.  I would rename the file name.  Any preference?

Reply:
I agree that it should match. I would name the file `tstQueryComparisonWithBoost` and test `QueryComparisonWithBoost`. Don't think we need `Geometry.Index` as that is implied.